src/ac-power/ac-power.c

```diff
@@ -12,21 +12,19 @@ static void help(void) {
                "Report whether we are connected to an external power source.\n\n"
                "  -h --help             Show this help\n"
                "     --version          Show package version\n"
-               "  -v --verbose          Show state as text\n"
-               , program_invocation_short_name);
+               "  -v --verbose          Show state as text\n",
+               program_invocation_short_name);
 }
 
 static int parse_argv(int argc, char *argv[]) {
 
-        enum {
+        enum
+        {
```

Comment:
This is against our coding style.

Reply:
ok, i can change this in the settings...

Reply:
hmm, this seems to be a problem with clang-format. When used with 8 char space indent, this results into 
```
enum
{
... EN1,
```
otherwise if used BraceWrapping AfterEnum to true this results into:
```
enum { EN1,
```
which is probably undesirable either...

Reply:
I guess changing the indent from 8 characters to 4 characters is not an option?

Reply:
this one I would consider a biggie... we really should be able to indent enum properly, and our indentation style doesn't appear particularly outlandish to me. sounds like something to fix in llvm?

(and no, please no 4 ch indenting)

Reply:
ok, i will reach out to the clang people for that. 
I can also write a small script to the tools/ folder, which fixes this as long as the fix from clang-format is not yet in the common package.
Maybe: tools/post-clang-format.php